src/sys/linux/random.cr

```diff
@@ -30,6 +30,21 @@ module Sys
       end
     end
 
+    # Returns a random UInt32.
+    def self.random_number : UInt32
```

Comment:
Somewhat. You avoid calling `next_u` twice when generating a 64bits number but always read *at least* a `next_u` (i.e. 4 bytes) when the maximum value fits.

Merged.